src/ecs/commands/runCommandInContainer.ts

```diff
@@ -187,3 +146,58 @@ async function verifyCliAndPlugin(window: Window): Promise<boolean> {
     }
     return true
 }
+
+async function isDeploymentInProgress(node: EcsContainerNode): Promise<boolean> {
+    const deployments = (await node.ecs.describeServices(node.clusterArn, [node.serviceName]))[0].deployments
+    if (deployments) {
+        for (let i = 0; i < deployments.length; i++) {
+            if (deployments[i].status === 'ACTIVE' || deployments[i].rolloutState === 'IN_PROGRESS') {
```

Comment:
just confirming: status=ACTIVE means "might be deploying"? That seems odd.

Reply:
> ACTIVE
    A service deployment that still has running tasks, but are in the process of being replaced with a new PRIMARY deployment. 

So there is definitely a deployment in progress, but the tasks are potentially being inactivated or spinning up. I'll change the wording to show a deployment is still active.

Reply:
jetbrains toolkit only checks IN_PROGRESS: https://github.com/aws/aws-toolkit-jetbrains/blob/0b0b551211e593756161c6e5016b5b8de82af3fc/jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/EcsExecUtils.kt#L204

maybe let them know if you found that status=ACTIVE is also needed.

Reply:
Informed @manodnyab since she piloted that feature.